BackofficeBundle/DependencyInjection/Configuration.php

```diff
@@ -22,13 +35,33 @@ public function getConfigTreeBuilder()
 
         $rootNode->children()
             ->arrayNode('front_languages')
-                ->info('Add the language available for the front with the key')
+                ->info('Add the language available for the front with the key, default (en, fr, de)')
```

Comment:
It could be the occasion to rephrase this help:
``Add the availables languages in the Front Office``